kubeconfig.go

```diff
@@ -71,8 +71,9 @@ func (k *KubeConfig) NewKubeConfigForRESTConfig(ctx context.Context, config *res
 			{
 				Name: fmt.Sprintf("%s-context", clusterName),
 				Context: KubeconfigContext{
-					Cluster: clusterName,
-					User:    fmt.Sprintf("%s-user", clusterName),
+					Cluster:   clusterName,
+					Namespace: namespace,
```

Comment:
I didn't add validation for Namespace since it should be default as a blank string.